static/js/tutorial.js

```diff
@@ -344,12 +344,9 @@ function finale(skip) {
     });
     deferred_work = [];
 
-    // We start you in a narrow so it's not overwhelming.
-    var newbie_stream = stream_data.get_newbie_stream();
-
-    if (newbie_stream) {
-        narrow.activate([{operator: "stream", operand: newbie_stream}]);
-    }
+    var sender_bot = "welcome-bot@zulip.com";
```

Comment:
@rishig I am not sure why it is not working for you. If you create a new user in any realm and login for the first time it is supposed to work. It's working for me.
Can you add a console.log near that statement and see whether it's executing or not. Also see whether `narrow.by('pm-with', sender_bot, {select_first_unread: true, trigger: 'sidebar'});` is working by entering in the js console. That can hep me debug the problem.